codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddS3ControlDependency.java

```diff
@@ -58,6 +59,19 @@ public List<String> runAfter() {
         );
     }
 
+    @Override
+    public void addConfigInterfaceFields(TypeScriptSettings settings, Model model, SymbolProvider symbolProvider, TypeScriptWriter writer) {
```

Comment:
The PR adds the NodeConfigLoader to support the `useArnRegion` configuration property of S3Control. 

Currently, only S3 has config loader support for this parameter, but the parameter is in both service models.